code/modules/antagonists/heretic/items/eldritch_flask.dm

```diff
@@ -1,8 +1,104 @@
 // An unholy water flask, but for heretics.
-// Heals heretics, hearms non-heretics. Pretty much identical.
+// Heals heretics, harms non-heretics. Pretty much identical.
 /obj/item/reagent_containers/cup/beaker/eldritch
 	name = "flask of eldritch essence"
 	desc = "Toxic to the closed minded, yet refreshing to those with knowledge of the beyond."
 	icon = 'icons/obj/antags/eldritch.dmi'
 	icon_state = "eldritch_flask"
 	list_reagents = list(/datum/reagent/eldritch = 50)
+
+// Unique bottle that lets you instantly draw blood from a victim
+/obj/item/reagent_containers/cup/phylactery
+	name = "phylactery of damnation"
+	desc = "Used to steal blood from your soon to be victims."
```

Comment:
```suggestion
	desc = "Used to steal blood from your soon-to-be victims."
```

I would put this behind get_examine_text(). Someone looting the corpse of a heretic probably shouldn't have this as their witnessed description